gh-badge.spec.js

```diff
@@ -1,61 +1,53 @@
-var assert = require('assert');
-var cproc = require('child_process');
-var isPng = require('is-png');
-var isSvg = require('is-svg');
+const assert = require('assert');
+const isPng = require('is-png');
+const isSvg = require('is-svg');
+const {spawn} = require('child_process');
 
 describe('The CLI', function () {
-
-  it('should provide a help message', function(done) {
-    var child = cproc.spawn('node', ['gh-badge.js']);
-    var buffer = '';
-    child.stdout.on('data', function(chunk) {
-      buffer += ''+chunk;
-    });
-    child.stdout.on('end', function() {
+  it('should provide a help message', function (done) {
+    const child = spawn('node', ['gh-badge.js']);
+    let buffer = '';
+    child.stdout.on('data', chunk => { buffer += ''+chunk; });
+    child.stdout.on('end', () => {
       assert(buffer.startsWith('Usage'));
       done();
     });
   });
 
-  it('should produce default badges', function(done) {
-    var child = cproc.spawn('node',
-      ['gh-badge.js', 'cactus', 'grown']);
-    child.stdout.once('data', function(chunk) {
-      var buffer = ''+chunk;
+  it('should produce default badges', function (done) {
+    const child = spawn('node', ['gh-badge.js', 'cactus', 'grown']);
+    child.stdout.once('data', chunk => {
+      const buffer = ''+chunk;
```

Comment:
The one other comment I had was to generalise this `spawn` + `stdout.once` + `buffer` code so this code isn't repeated in every test - Something line
```
function run(args, cb) {
  const child = spawn('node', ['gh-badge.js', ...args]);
  child.stdout.once('data', chunk => {
    const buffer = '' + chunk;
    cb(buffer);
  });
}
```

Then here you'd do:
```
it('should produce default badges', function (done) {
  run(['cactus', 'grown'], buffer => {
    assert.ok(isSvg(buffer));
    // ... etc
  });
```